simplq/src/components/common/Modal/Modal.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Modal from '.';
+
+describe('Modal', () => {
+  it('should render the modal if it is open', () => {
+    const { getByText } = render(<Modal open>Hello</Modal>);
+    getByText('Hello');
+  });
+
```

Comment:
Kudos for adding tests! It's about time we think of incorporating unit tests at least some basic integration tests.